### PR TITLE
fix: float of slider handle for RTL support

### PR DESCRIPTION
**Pull request by shaveko**
related to https://oat-sa.atlassian.net/browse/TR-2757

### Description

This PR fixes the styling for noUI slider used with SliderInteraction

To test you need to apply changes to `taoQtiItem/views/node_modules/@oat-sa/tao-item-runner-qti` from this brach `feat/TR-2757/slider-interaction-rtl`

🚩**NOTE** : publish and update dependency for `tao-item-runner-qti-fe` before meging

### CODE REVIEW

File: views/scss/inc/_jquery.nouislider.scss [hunk 1]
```diff
@@ -5,250 +5,250 @@
...
+  position: absolute;
+  right: 0;
+  top: 0;
+  left: 0;
+  bottom: 0;
 }
 .noUi-handle {
-    position: relative;
-    z-index: 1;
+  float: left;
```
Comment (marpesia) [inline views/scss/inc/_jquery.nouislider.scss:29]: Hi, I can't see any effect adding this style. I suggest removing it if it is not necessary.

**Review by marpesia — CHANGES_REQUESTED** [review]

File: views/scss/inc/_jquery.nouislider.scss (hunk 1 above)
Comment (shaveko) [reply views/scss/inc/_jquery.nouislider.scss:29]: Sorry, may be I'm asking something very stupid: did you rebuild css after you remove this style from this scss?

**Review by marpesia — APPROVED** [review]
LGTM

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x]  Commits messages are meaningful
- [x]  Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: views/scss/inc/_jquery.nouislider.scss (hunk 1 above)
Comment (meefox) [reply views/scss/inc/_jquery.nouislider.scss:29]: I also didn't get the point for float here, 
it is a relative element that is surrounded by nothing. 
Or may I miss the point?

**Review by meefox — CHANGES_REQUESTED** [review]

File: views/scss/inc/_jquery.nouislider.scss [hunk 2]
```diff
@@ -5,250 +5,250 @@
...
+      &:before {
+        width: 1px;
+        height: 5px;
+        position: absolute;
+        content: "";
+        background: whiten($textColor, 0.15);
+        left: 50%;
+      }
+    }
+    &:before {
```
Comment (meefox) [inline views/scss/inc/_jquery.nouislider.scss:189]: does it make any sense?

File: views/scss/inc/_jquery.nouislider.scss (hunk 1 above)
Comment (shaveko) [reply views/scss/inc/_jquery.nouislider.scss:29]: > I also didn't get the point for float here, it is a relative element that is surrounded by nothing. Or may I miss the point?

here the main purpose of `float` is to specify the positioning inside the container, which is wrong for slider's logic if `rtl` direction is applied

File: views/scss/inc/_jquery.nouislider.scss [hunk 3]
```diff
@@ -5,250 +5,250 @@
...
 .noUi-target,
 .noUi-target * {
-    -webkit-touch-callout: none;
-    -webkit-user-select: none;
-    -ms-touch-action: none;
-    -ms-user-select: none;
-    -moz-user-select: none;
-    -moz-box-sizing: border-box;
-    box-sizing: border-box;
+  -webkit-touch-callout: none;
```
[the remark this answers is not in the document]
Comment (shaveko) [reply views/scss/inc/_jquery.nouislider.scss]: done

**Review by meefox — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**